src/test/java/org/verapdf/io/ASMemoryInStreamTest.java

```diff
@@ -0,0 +1,216 @@
+/**
+ * This file is part of veraPDF Parser, a module of the veraPDF project.
+ * Copyright (c) 2023, veraPDF Consortium <info@verapdf.org>
+ * All rights reserved.
+ *
+ * veraPDF Parser is free software: you can redistribute it and/or modify
+ * it under the terms of either:
+ *
+ * The GNU General public license GPLv3+.
+ * You should have received a copy of the GNU General Public License
+ * along with veraPDF Parser as the LICENSE.GPL file in the root of the source
+ * tree.  If not, see http://www.gnu.org/licenses/ or
+ * https://www.gnu.org/licenses/gpl-3.0.en.html.
+ *
+ * The Mozilla Public License MPLv2+.
+ * You should have received a copy of the Mozilla Public License along with
+ * veraPDF Parser as the LICENSE.MPL file in the root of the source tree.
+ * If a copy of the MPL was not distributed with this file, you can obtain one at
+ * http://mozilla.org/MPL/2.0/.
+ */
+package org.verapdf.io;
+
+import org.junit.Test;
+import org.verapdf.as.io.ASInputStream;
```

Comment:
Codacy found unused import `org.verapdf.as.io.ASInputStream;`

Reply:
Fixed in https://github.com/veraPDF/veraPDF-parser/compare/e856d5f6bf1004dcf51653e72f2bd2be197feca4..2ac982977e389e7bf2defea03b641704357477bc